src/types/SelectValues.ts

```diff
@@ -0,0 +1,5 @@
+export enum SelectValues {
+  all = 'all',
+  active = 'active',
+  completed = 'completed',
+}
```

Comment:
```suggestion
export enum SelectValues {
  All = 'all',
  Active = 'active',
  Completed = 'completed',
}
```
use CamelCase for enum keys